ChangeLog

```diff
@@ -1,8 +1,12 @@
+2015-03-15  Kevin Murray  <spam@kdmurray.id.au>  &  Titus Brown  <titus@idyll.org>
```

Comment:
On Sat, Mar 14, 2015 at 03:57:28PM -0700, Kevin Murray wrote:
> > @@ -1,8 +1,12 @@
> > +2015-03-15  Kevin Murray  <spam@kdmurray.id.au>  &  Titus Brown  <titus@idyll.org>
> 
> @mr-c  @ctb  is this the right way to credit 2 people in a CL entry?

Oh, no need to credit... seriously... :)